The minigzip code does the same thing as sys-libs/zlib[minizip] but for .gz files rather than .zip

It uses a macro 'OF(args)' - this seems to be for compatibility with old C compilers.
It is declared in zconf.h but newer versions of zlib use _Z_OF instead.

This patch has been submitted upstream at:
https://github.com/mantidproject/mantid/pull/149

diff --git a/MantidPlot/src/zlib123/minigzip.c b/MantidPlot/src/zlib123/minigzip.c
index 97fbae1..5f4612f 100644
--- a/MantidPlot/src/zlib123/minigzip.c
+++ b/MantidPlot/src/zlib123/minigzip.c
@@ -50,6 +50,15 @@
 #  include <unix.h> /* for fileno */
 #endif
 
+/* New versions of zlib use _Z_OF rather than OF */
+#ifndef OF
+#  ifdef _Z_OF
+#    define OF _Z_OF
+#  else
+#    define OF(args) args
+#  endif
+#endif
+
 #ifndef WIN32 /* unlink already in stdio.h for WIN32 */
   extern int unlink OF((const char *));
 #else
